src/main/scala/piconot/external/Empty.bot

```diff
@@ -0,0 +1,20 @@
+
+# State 0: move left
+State 0:
+,← => ←,0
+←, => .,1
+
+# State 1: move up
+State 1:
+,↑ => ↑,1
+↑,↓ => ↓,2
+
+# State 2: fill this column to the bottom
+State 2:
+,↓ => ↓,2
+↓,→ => →,3
+
+# State 3: fill this column to the top
+State 3:
+,↑ => ↑,3
+↑,→ => →,2
```

Comment:
I really like this. You can just about read it like plain English. The one thing I don't like is lines starting with a comma -- there's something just kind of strange about that. However, once you get used to that, it's really quite easy to read!